include/zephyr/arch/cache.h

```diff
@@ -335,6 +335,26 @@ size_t arch_icache_line_size_get(void);
 
 #endif /* CONFIG_ICACHE || __DOXYGEN__ */
 
+#if CONFIG_CACHE_COHERENCY  || __DOXYGEN__
+bool arch_cache_is_ptr_cached(void *ptr);
+
+#define cache_is_ptr_cached(ptr) arch_cache_is_ptr_cached(ptr)
+
+
+bool arch_cache_is_ptr_uncached(void *ptr);
+
+#define cache_is_ptr_uncached(ptr) arch_cache_is_ptr_uncached(ptr)
```

Comment:
remove the space between `bool` and `define` (like it is done in the following lines).